lib/nnue_training_data_formats.h

```diff
@@ -7682,7 +7688,7 @@ namespace binpack
         CompressedTrainingDataFile m_inputFile;
         std::atomic_int m_numRunningWorkers;
 
-        static constexpr int threadBufferSize = 256 * 256 * 4;
+        static constexpr int threadBufferSize = 256 * 256 * 16;
```

Comment:
IIRC the size of a single entry is ~200B, that means each reading thread will take about 200MB with this. I guess this is fine

Reply:
yes, I think that's fine.